### ISSUE TITLE
Feature list 'audio' is required but could not be found. 

### ISSUE THREAD

**Issue by yongxuUSTC** [antoine77340/Youtube-8M-WILLOW#5]
Hi i want to use your code to do experiments on audioset (audio only, frame-level features 128-dim). 

But i have problem with your NetVLADModelLF model:
**Command:** 
python train.py --train_data_pattern='/vol/vssp/msos/yx/audioset/audioset_v1_embeddings/bal_train/*.tfrecord' --model=NetVLADModelLF --train_dir=gatednetvladLF-256k-1024-80-0002-300iter-norelu-basic-gatedmoe --frame_features=True --feature_names='audio_embedding' --feature_sizes='128' --batch_size=80 --base_learning_rate=0.0002 --netvlad_cluster_size=256 --netvlad_hidden_size=128 --moe_l2=1e-6 --iterations=300 --learning_rate_decay=0.8 --netvlad_relu=False --gating=True --moe_prob_gating=True --max_step=700000

**Error:** 2017-07-24 15:44:44.093022: W tensorflow/core/framework/op_kernel.cc:1158] Invalid argument: Name: , Feature list 'audio' is required but could not be found.  Did you mean to include it in feature_list_dense_missing_assumed_empty or feature_list_dense_defaults?

_But i can successfully run using LstmModel or GruModel :_ 
**command:** 
python train.py --train_data_pattern='/vol/vssp/msos/yx/audioset/audioset_v1_embeddings/bal_train/*.tfrecord' --frame_features=True --model=LstmModel --feature_names='audio_embedding' --feature_sizes='128' --train_dir=tmp_model/frame_level_lstm_model_bal

The feature name in Audioset is **"--feature_names='audio_embedding''**, i think there might be feature name error in your NetVLADModelLF definition. Could you give me some help on it ?

Thanks a lot.

**Comment by antoine77340** [comment]
Hi, it is a good idea to apply the model on the audioset. 
However, as opposed to the LSTM or GRU model, the code of the netvlad based model was not
supposed to be run without the video (or without the audio). You need to make some small modification on the frame_level_models.py code. More specifically, you just need to modify the NetVLADModelLF class, by changing the architecture so it only has the audio branch (without the video branch).
I am sorry I do not have much time to upload new code for it, but I do not think this modification is very difficult to do :).

**Comment by punitagrawal32** [comment]
@yongxuUSTC @antoine77340 Were you able to make the changes? I am facing a similar issue, and I request you to share the changes you made to the 'frame_level_models.py' to get the code running for the audio features only.

**Comment by punitagrawal32** [comment]
@yongxuUSTC @antoine77340 Awaiting a response from you. 

Thank you.
